functions/serverData.js

```diff
@@ -53,6 +59,12 @@ module.exports = {
 			return client.memory[guildIdString];
 		}
 	},
+	get getMemory() {
+		return this._getMemory;
+	},
+	set getMemory(value) {
```

Comment:
why did you use the get/set keywords? I really don't think this is what you meant to do